features/scroll-buttons.yml

```diff
@@ -0,0 +1,9 @@
+name: ::scroll-button
+description: The `::scroll-button(<dir>)` CSS pseudo-element is a focusable button which scrolls its originating scroll container element in the specified direction.
+spec: https://drafts.csswg.org/css-overflow-5/#scroll-buttons
+group: scrolling
+caniuse: css-scroll-buttons
+status:
+  compute_from: css.selectors.scroll-button
+compat_features:
+  - css.selectors.scroll-button
```

Comment:
Let's remove all of this if the feature isn't yet covered by BCD.
If the implementation of this feature is shipping enabled then let's pause this PR, add the corresponding BCD keys (or check if the BCD collector can do it automatically), and then add the correct BCD key(s) here.
If not (and [the chromestatus entry](https://chromestatus.com/feature/5093129273999360) seems to say this isn't yet enabled), then let's leave this out for now.